src/sentry/static/sentry/app/views/settings/components/text/textBlock.jsx

```diff
@@ -0,0 +1,8 @@
+import styled from 'react-emotion';
+
+const TextBlock = styled.div`
```

Comment:
why not a styled `<p/>`?

Reply:
Since semantics matter less in this world, I was hoping to avoid the nested `<p>` problem you brought up when we discussed IRL by just styling a div.  I can style the `<p>` if you feel strongly about it.

Reply:
That's a pretty convincing let's do it!